lib/manageiq/environment.rb

```diff
@@ -9,16 +9,16 @@ def self.manageiq_plugin_setup(plugin_root = nil)
       # determine plugin root dir. Assume we are called from a 'bin/' script in the plugin root
       plugin_root ||= Pathname.new(caller_locations.last.absolute_path).dirname.parent
 
-      manageiq_plugin_update(plugin_root)
+      manageiq_plugin_update(plugin_root, force: false)
     end
 
-    def self.manageiq_plugin_update(plugin_root = nil)
+    def self.manageiq_plugin_update(plugin_root = nil, force: true)
       # determine plugin root dir. Assume we are called from a 'bin/' script in the plugin root
       plugin_root ||= Pathname.new(caller_locations.last.absolute_path).dirname.parent
 
       setup_gemfile_lock if ENV["CI"]
       install_bundler(plugin_root)
-      bundle_update(plugin_root)
+      bundle_update(plugin_root, force: force)
```

Comment:
here `force` is fine, because it's a parameter to the method named `bundle_update`